wordpress-mobile/WordPress-iOS
# UL&S: SignupMagicLinkViewController: Open Mail button

Ref. #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/379

### To test
Test this on a device to see the different email apps appear in the action sheet.

1. Enable `.unifiedSignup` feature flag
2. Check out the Auth PR branch
3. Set the podfile to develop locally
4. In Authenticator, go to `UnifiedSignupViewController` and set a testing email address
5. `rake dependencies`
6. Sign up for WordPress.com > Sign up with Email > Send Link by Email. **Expected:** you are navigated to the next screen.
7. Tap the "Open Mail" button. **Expected:** an action sheet with your email apps appears. Select one, open the magic link email and tap the link.
8. You should be navigated to the Signup Epilogue screen that confirms the display name, username, and (optional) password. 

### Screenshots
| Light mode | iPhone 11 Pro |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/90436900-1417a980-e097-11ea-91bb-e20694d3f2d2.PNG"><img src="https://user-images.githubusercontent.com/1062444/90436900-1417a980-e097-11ea-91bb-e20694d3f2d2.PNG" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/90436940-21349880-e097-11ea-96ce-81c2ebc41211.PNG"><img src="https://user-images.githubusercontent.com/1062444/90436940-21349880-e097-11ea-96ce-81c2ebc41211.PNG" width="350" /></a> |
| <a href="https://user-images.githubusercontent.com/1062444/90436972-2e518780-e097-11ea-8bcb-c7e73bb2f6fe.PNG"><img src="https://user-images.githubusercontent.com/1062444/90436972-2e518780-e097-11ea-8bcb-c7e73bb2f6fe.PNG" width="350" /></a> |   | 

PR submission checklist:
